### ISSUE TITLE
Documenting constants

### ISSUE THREAD

**Issue by timholy** [MichaelHatherly/Docile.jl#153]
```julia
julia> module MyModule

       using Docile

       const x = 5

       @doc "whoops!" -> x

       end

ERROR: Unknown category for '@doc'ed object.
 in lateguess at /home/tim/.julia/v0.3/Docile/src/Legacy/atdoc.jl:68
```
(works in 0.4)

**Comment by MichaelHatherly** [comment]
Sorry for the slow response on this one @timholy.

Adding this could be done relatively simply I think, though it can't just be backported from ``Base`` since things have diverged quite a bit over the last few months.

Using bare docstrings instead should work fine here I think, and that's probably the preferred route to take, given that they've been added to ``Base``, instead of using ``@doc`` directly. Also, documenting constants after definition like in this example seem to be quite rarely needed, have you got a specific place this is needed?

**Comment by timholy** [comment]
All the issues I've reported stem from trying to support 0.3 in my packages. Bare docstrings aren't available on 0.3.

But given that 0.4 should be out soon, I'm just contemplating making `@doc` a no-op on my 0.3 packages. So don't worry about this.

**Comment by MichaelHatherly** [comment]
Bare docstrings can be picked up by Docile in 0.3 without Docile being imported into the package, so you should be able to just add bare docstrings as usual and they'll be picked up by Docile/Lexicon if a user happens to import your packages and also does ``using Lexicon``.
